src/analytics/selectors.ts

```diff
@@ -116,6 +127,16 @@ export const getCurrentUserTraits = createSelector(
       }
     }
 
+    const hasTokenBalanceFields: Record<string, boolean> = {
+      hasTokenBalance: tokensWithBalance.length > 0,
+      ...Object.fromEntries(
+        networkIds.map((networkId) => [
+          `has${toPascalCase(networkId)}TokenBalance`,
+          tokensWithBalance.some((token) => token.networkId === networkId),
+        ])
+      ),
```

Comment:
I was going back and forth with one property per network vs a single property listing all network ids with balances (e.g., `networkIdsWithBalance: celo-mainnet, ethereum-mainnet`) but went with the former as it seems easier to handle booleans over csv strings in analytics tools, and also aligns with the chain segmentation for `totalBalanceUsd` where we will need one prop per chain. But could be convinced otherwise.

Reply:
I think i prefer this way too for ease of working with in analytics tools